pkgs/tools/security/saml2aws/default.nix

```diff
@@ -2,7 +2,7 @@
 
 buildGoPackage rec {
   name = "saml2aws-${version}";
```

Comment:
you can also update to `pname = "saml2aws"`

Reply:
@teto, sorry I got distracted and didn't finish. I will now.

Reply:
and change the commit title to match nixpkgs guidlines, such as:
`saml2aws: X -> Y`

Reply:
Everything is done.

Reply:
Tested and working.